data/connection.go

```diff
@@ -166,6 +167,14 @@ func (c *PostgresSQL) GetOrders(userID int, orderID *int) (model.Orders, error)
 
 			if len(coffee) > 0 {
 				orders[n].Items[i].Coffee = coffee[0]
+
+				ing := []model.CoffeeIngredient{}
+				err := c.db.Select(&ing, "SELECT ingredient_id FROM coffee_ingredients WHERE coffee_id=$1", orders[n].Items[i].Coffee.ID)
```

Comment:
do we want to filter on ingredients where quantity > 0?

```suggestion
				err := c.db.Select(&ing, "SELECT ingredient_id FROM coffee_ingredients WHERE coffee_id=$1 AND quantity > 0", orders[n].Items[i].Coffee.ID)
```